.zuul.yaml

```diff
@@ -0,0 +1,28 @@
+- project:
+    name: kubernetes-sigs/cluster-api-provider-openstack
+    check:
+      jobs:
+        - cluster-api-provider-openstack-conformance-stable
+
+- job:
+    name: cluster-api-provider-openstack-conformance-stable
+    parent: init-test
+    timeout: 36000
+    description: |
+      Test CAPO & Kubernetes master against CAPI & CABPKG stable
+    run: .zuul/playbooks/run.yaml
+    nodeset: ubuntu-xenial-arm64
```

Comment:
This job runs on arm64? If not, please use `ubuntu-xenial-citynetwork`

Reply:
Done